fem/pgridfunc.cpp

```diff
@@ -539,6 +539,7 @@ void ParGridFunction::SaveAsOne(std::ostream &out)
    int *nfdofs = new int[NRanks];
    int *nrdofs = new int[NRanks];
 
+   data.ReadWrite(MemoryClass::HOST,data.Capacity());
```

Comment:
Why not just call `HostRead()`? Also do we need `ReadWrite`? Isn't `Read` sufficient?

Reply:
Good point.